internal/services/loadbalancer/loadbalancer_resource_test.go

```diff
@@ -355,6 +355,7 @@ resource "azurestack_lb" "test" {
   name                = "acctestlb-%d"
   resource_group_name = azurestack_resource_group.test.name
   location            = azurestack_resource_group.test.location
+  sku                 = "Basic"
```

Comment:
we will need to add a test for the standard lb

Reply:
Just tested for standard, it seems not supported, so I will remove the value from the validation.